### DISCUSSION TITLE
Missing icons for downloads, public and templates in bloom theme.

### DISCUSSION THREAD

**Discussion by arcolinuxz** [linuxdeepin/developer-center#3023] [Features Request & Ideas | 特性请求 & 头脑风暴]
On thunar we are missing icons in the bloom theme. The other icon themes miss them too.

Solution 1: make links to standard folder for now
Solution 2: make new icons for them

![iconsmissing](https://user-images.githubusercontent.com/36679406/82869611-d4757180-9f2e-11ea-9fda-a8a557893f52.png)
The wish of a "Deepin promoter".
https://arcolinuxd.com/category/arcolinuxd/deepin/

**Comment by justforlxz** [comment]
I assign the issue to the community product manager, who submits it to the design team to supplement the corresponding icon.

**Comment by meiyixiang** [comment]
I have recorded this suggestion. Thank you for your feedback
